winml/test/common/taefTestMacros.h

```diff
@@ -48,14 +50,17 @@ using namespace WEX::TestExecution;
 
 #ifndef USE_DML
 #define GPUTEST \
-  WINML_SUPRESS_UNREACHABLE_BELOW(WINML_SKIP_TEST("GPU tests disabled because this is a WinML only build (no DML)"))
+  WINML_SKIP_TEST("GPU tests disabled because this is a WinML only build (no DML)")
+#define GPUTEST_ENABLED alwaysFalse()
 #else
 #define GPUTEST                                                                             \
   bool noGPUTests;                                                                          \
   if (SUCCEEDED(RuntimeParameters::TryGetValue(L"noGPUtests", noGPUTests)) && noGPUTests) { \
     WINML_SKIP_TEST("This test is disabled by the noGPUTests runtime parameter.");          \
     return;                                                                                 \
   }
+#define GPUTEST_ENABLED bool noGPUTests; \
```

Comment:
Since we are touching this file, can we conform all the new/existing variable names to snake_case?